pjt00/tclone/views.py

```diff
@@ -23,3 +32,37 @@ def form_valid(self, form):
 
 class EntryOKView(TemplateView):
     template_name = 'tclone/entryok.html'
+
+class FollowView(LoginRequiredMixin, CreateView):
+    model = FF
+    fields = []
+    template_name = 'tweet/profile.html'

```

Comment:
```suggestion
    template_name = 'tclone/follow.html'
```
前回のレビュー時に、使っていないTemplateは削除しましょうとお伝えしましたが、follow.htmlを削除してしまうと、エラーメッセージが表示されませんので、残しておいてください。

Reply:
[**4.2.2**]で修正しました。`profile.html`のほうにもフォームがあるのですが、これだとエラーが吐けないのでしょうか？

Reply:
`profile.html`でもエラーの表示はできますが、この時FollowViewから`profile.html`にprofileViewと同じコンテキストを渡せるようにしなければなりません。
また`template_name='tclone/follow.html'`にしてください。
修正する時は、変更箇所が正しく表示されるかの確認をしておきましょう。

Reply:
[**4.3.1**]で修正しました。凡ミスです…